api/questions/views.py

```diff
@@ -0,0 +1,88 @@
+    """
+    View class for performing CRUD operations on questions
+    """
+    permission_classes = [permissions.IsAuthenticated]
+
+    def post(self, request, id=None):
+        """
+        Endpoint for posting questions
+        """
+        queryset = Meetup.objects.all()
+        request.data['created_by'] = request.user
+
+        serializer = QuestionsSerializer(data=request.data)
+        if serializer.is_valid():
+            Question.objects.update_or_create(
+                title=request.data.get('title'),
+                body=request.data.get('body'),
+                defaults={
+                    'meetup': get_object_or_404(queryset, id=id),
+                    'created_by': request.data.get('created_by')
+                }
+            )
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors)
+
+class QuestionEditViews(APIView):
+    """
+    class to edit question posted
+    """
+    permission_classes = [permissions.IsAuthenticated]
+
+    def put(self, request, id=None):
+        request.data['created_by'] = request.user
+        is_authenticated = request.user
+        owner = Question.objects.filter(created_by_id = is_authenticated).first()
+
+        if owner:
+            obj = Question.objects.get(id=id)
+            print(obj)
+            serializer = QuestionsSerializer(obj, data=request.data)
+            if serializer.is_valid():
+                Question.objects.filter(id=id).update(
+                    title=request.data.get('title'),
+                    body=request.data.get('body')
+                )
+                return Response(serializer.data, status=status.HTTP_201_CREATED)
+            return Response(serializer.errors)
+        return Response({
+            "error": "You are not the owner of this question"
+        }, status=status.HTTP_401_UNAUTHORIZED)
+    
+    def delete(self, request, id=None):
+        try:
+            request.data['created_by'] = request.user
+            is_authenticated = request.user
+            owner = Question.objects.filter(created_by_id = is_authenticated).first()
+
+            if owner:
+                question = Question.objects.filter(id=id).first()
+                if question:
+                    question.delete()
+                    return Response({
+                        "message": "Question deleted successfully"
+                    }, status=status.HTTP_200_OK)
+                return Response({
+                    "error": "The question does not exist in the database"
+                }, status=status.HTTP_404_NOT_FOUND)
+            return Response({
+                "error": "You are not the owner of this question"
+            }, status=status.HTTP_401_UNAUTHORIZED)
+        except ValidationError as e:
+            message = str(e)
+            return Response({
+                "error": message
+            }, status=status.HTTP_404_NOT_FOUND)
```

Comment:
Ensure there is a blank line at the end of the code in order to conform to pep8 standards